docs/content/preview/secure/encryption-at-rest.md

```diff
@@ -22,38 +22,38 @@ This page describes how to enable and disable encryption at rest in a YugabyteDB
 
 ### Step 1. Create encryption key
 
-First, you will generate the universe key data. This data can have length 32, 40, or 48. Larger keys are slightly more secure with slightly worse performance. Run the following on your local filesystem.
+First, you will generate the universe key data. This data can have length 32, 40, or 48. Larger keys are more secure with slightly worse performance. Run the following on your local filesystem.
 
 ```sh
-$ openssl rand -out universe_key [ 32 | 40 | 48 ]
+$ openssl rand -out /path/to/universe_key [ 32 | 40 | 48 ]
 
 ```
 
 ### Step 2. Copy key to master nodes
 
-In this example, assume a 3 node RF=3 cluster with addresses ip1, ip2, ip3.
-Copy the universe key onto each master filesystem, in the same location on every node.
+In this example, assume a 3 node RF=3 cluster with `MASTER_ADDRESSES=ip1:7100,ip2:7100,ip3:7100`. Choose any string <key_id> for this key and use yb-admin to copy the key to each of the masters.
 
 ```sh
-$ for ip in ip1 ip2 ip3
-  do
-    scp -i <ssh_key> universe_key ip:/mnt/d0/yb-data/master
-  done
+$ yb-admin -master_addresses $MASTER_ADDRESSES add_universe_keys_to_all_masters \
```

Comment:
@rahuldesirazu please verify this command is correct. I added the continuation on line 37 as I was removing the hard-wrapped lines all over the place.